### ISSUE TITLE
Update version for terraform-google-modules/vm/google//modules/instance_template

### ISSUE THREAD

**Issue by q2w** [terraform-google-modules/terraform-google-lb-http#440]
### TL;DR

The current version of terraform-google-modules/vm/google//modules/instance_template used in examples references to centos-7 as default source_image. https://github.com/terraform-google-modules/terraform-google-vm/blob/v9.0.0/modules/instance_template/variables.tf#L113

The support for centos-7 has ended on GCP.  https://cloud.google.com/compute/docs/eol/centos-eol-guidance

All the test using `terraform-google-modules/vm/google//modules/instance_template` are failing. 

### Expected behavior

The mig in the example should be created without error.

### Observed behavior

The mig in the example is not getting created. 

### Terraform Configuration

```hcl
https://github.com/terraform-google-modules/terraform-google-lb-http/blob/master/examples/cdn-policy/main.tf
```

### Terraform Version

```sh
Terraform v1.7.5
on linux_amd64
+ provider registry.terraform.io/hashicorp/google v5.38.0
+ provider registry.terraform.io/hashicorp/google-beta v5.38.0
+ provider registry.terraform.io/hashicorp/random v3.6.2

Your version of Terraform is out of date! The latest version
is 1.9.3. You can update by downloading from https://www.terraform.io/downloads.html
```
